### PR TITLE
Fix size of portrait images with the SIZE_NORMAL setting.

### PR DESCRIPTION
**Pull request by toger5**
With this change, portrait images are drawn with the same width than Landscape images.
This was the [requested behavior from design.](https://github.com/matrix-org/matrix-react-sdk/pull/7017#issuecomment-957761011).
This PR https://github.com/matrix-org/matrix-react-sdk/pull/7171 fixed a bug for really high images but also made portrait images  not behave as desired anymore.
(see the comments in the changed files for the logic used to determine image sizes)
Left old, right new (for the large setting nothing changes)
image

---
This change is marked as an *internal change* (Task), so will not be included in the changelog.

Preview: https://619f6fe0dafa3d153dcf4a2f--matrix-react-sdk.netlify.app
⚠️ Do you trust the author of this PR? Maybe this build will steal your keys or give you malware. Exercise caution. Use test accounts.

### CODE REVIEW

File: src/settings/enums/ImageSize.ts [hunk 1]
```diff
@@ -14,8 +14,13 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
+// For Large the image gets drawn as big as possible.
+// constraint by: timeline width, manual heigh overrides, SIZE_LARGE.h
```
Comment (turt2live) [inline src/settings/enums/ImageSize.ts:18]:
```suggestion
// constraint by: timeline width, manual height overrides, SIZE_LARGE.h
```

File: src/settings/enums/ImageSize.ts [hunk 2]
```diff
@@ -14,8 +14,13 @@ See the License for the specific language governing permissions and
...
 const SIZE_LARGE = { w: 800, h: 600 };
-const SIZE_NORMAL = { w: 324, h: 220 };
+
+// For Normal the image gets drawn to never exceed SIZE_NORMAL.w, SIZE_NORMAL.h
+// constraint by: timeline width, manual heigh overrides
```
Comment (turt2live) [inline src/settings/enums/ImageSize.ts:22]:
```suggestion
// constraint by: timeline width, manual height overrides
```

**Review by turt2live — APPROVED** [review]
The code looks fine, though I'm a bit sad to see portrait images taking up even more space again - it feels like it defeats the purpose of the smaller image size setting.

**Review by gaelledel — CHANGES_REQUESTED** [review]
@toger5 I'm sorry, there must have been a misunderstanding with this. My original request was to simply make sure that we have a set width for landscape and a set width for portrait whether it is image or video based media.

**Comment by toger5** [comment]
I revisited this PR and added another default value for portrait images. They are handled the same way (they are sized so that the width fits the requested width, EXCEPT if the height exceeds a maximum)  but just use different default values now.
 - this makes sure 16:9 images and 9:16 images take the same area in the timeline.
 - for images with an aspect ration lesser than 1:1 it introduces a new width so that the timeline is less chaotic.
 
left old, right new

**Review by gaelledel — COMMENTED** [review]
Perfect, thank you very much @toger5!

**Review by gaelledel — APPROVED** [review]

**Pull request merged**